skbeam/core/accumulators/projector.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+
+"""
+authors: Mikhail Dubrovin, TJ Lane, Christopher O'Grady
+"""
+
+
+class Projector(object):
```

Comment:
This is a very good question. I think we can put this at the accumulators directory for now. But it will be helpful if we can think about better structure to organize tools at skbeam, especially when the package is growing larger.